### PR TITLE
Rebuild packages for missing metadata vol. 10

### PR DESCRIPTION
**Pull request by Chocimier**
All packages was build (when available) for x86_64, x86_64-musl, armv7hf and aarch64-musl and passed check stage on x86_64 and x86_64-musl. Nothing was tested manually whether it still works.

xdaliclock license is https://fedoraproject.org/wiki/Licensing:MIT#Old_Style, and not what spdx calls MIT

18 more packages remain to rebuild, mostly failing to build.

### CODE REVIEW

File: srcpkgs/crimson/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 pkgname=crimson
 version=0.5.3
-revision=1
+revision=2
 build_style=gnu-configure
-nocross=yes
 makedepends="SDL-devel SDL_mixer-devel SDL_net-devel SDL_ttf-devel"
 short_desc="Tactical war game in the tradition of the popular Battle Isle"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/crimson/template]: GPL-2.0-or-later

File: srcpkgs/dclock/template [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=dclock
 version=2.2.2
-revision=2
+revision=3
 hostmakedepends="pkg-config"
 makedepends="libXft-devel libXt-devel"
 short_desc="Digital clock for X"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/dclock/template]: GPL-2.0-or-later

File: srcpkgs/dialogbox/template [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'dialogbox'
 pkgname=dialogbox
 version=1.0
-revision=1
+revision=2
 build_style=qmake
 hostmakedepends="qt5-host-tools qt5-devel"
 makedepends="qt5-devel"
```
Comment (maxice8) [inline srcpkgs/dialogbox/template]: GPL-3.0-or-later

File: srcpkgs/gavl/template [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
 pkgname=gavl
 version=1.4.0
-revision=2
+revision=3
 build_style=gnu-configure
-nocross="yes"
 hostmakedepends="doxygen"
 short_desc="Low level library, upon which multimedia APIs can be built"
 maintainer="Logen K <logen@sudotask.com>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/gavl/template]: GPL-3.0-or-later

File: srcpkgs/gnome-shell-mousewheel-zoom/template [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
 version=0.8.0
-revision=1
+revision=2
+wrksrc=gnome-shell-mousewheel-zoom-upstream-0.8.0
+hostmakedepends="vala pkg-config"
+makedepends="libglib-devel libX11-devel"
+depends="gnome-shell"
 short_desc="Enable mousewheel zoom using left-alt key using gnome-shell"
-homepage="https://github.com/tobiasquinn/gnome-shell-mousewheel-zoom"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/gnome-shell-mousewheel-zoom/template]: GPL-3.0-or-later

File: srcpkgs/gengetopt/template [hunk 6]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'gengetopt'
 pkgname=gengetopt
 version=2.22.6
-revision=1
-disable_parallel_build=yes
+revision=2
 build_style=gnu-configure
 hostmakedepends="flex"
 makedepends="libfl-devel"
```
Comment (maxice8) [inline srcpkgs/gengetopt/template]: GPL-3.0-or-later

File: srcpkgs/gtkimageview/template [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 _githash=77abd2122c19d2eca21599404a2d3aece35081e2
 wrksrc="${pkgname}-${_githash}"
 build_style=gnu-configure
 configure_script="./autogen.sh"
-maintainer="Orphaned <orphan@voidlinux.eu>"
 hostmakedepends="gnome-common pkg-config gtk-doc libtool glib-devel"
 makedepends="gtk+-devel"
+short_desc="Widget that provides a zoomable and panable view of a GdkPixbuf"
+maintainer="Orphaned <orphan@voidlinux.eu>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/gtkimageview/template]: LGPL-2.1-or-later

File: srcpkgs/hardinfo/template [hunk 8]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'hardinfo'
 pkgname=hardinfo
 version=0.5.1
-revision=4
-lib32disabled=yes
+revision=5
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="gtk+-devel libsoup-devel desktop-file-utils"
```
Comment (maxice8) [inline srcpkgs/hardinfo/template]: GPL-2.0-or-later

File: srcpkgs/hdapsd/template [hunk 9]
```diff
@@ -1,13 +1,12 @@
...
 pkgname=hdapsd
 version=20141024
-revision=3
+revision=4
 only_for_archs="i686 x86_64 i686-musl x86_64-musl"
 build_style=gnu-configure
 short_desc="HDAPS userspace hard drive protection daemon"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/hdapsd/template]: GPL-2.0-or-later

File: srcpkgs/jfsutils/template [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'jfsutils'
 pkgname=jfsutils
 version=1.1.15
-revision=1
+revision=2
 build_style=gnu-configure
 makedepends="libuuid-devel"
 short_desc="JFS filesystem utilities"
```
Comment (maxice8) [inline srcpkgs/jfsutils/template]: GPL-2.0-or-later

File: srcpkgs/miruo/template [hunk 11]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=miruo
 version=0.9.6
-revision=1
+revision=2
 build_style=gnu-configure
 makedepends="libpcap-devel"
 short_desc="Pretty-print TCP session monitor/analyzer"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/miruo/template]: GPL-3.0-only

File: srcpkgs/mmv/template [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'mmv'
 pkgname=mmv
 version=1.01b
-revision=2
+revision=3
 wrksrc=${pkgname}-${version}.orig
 build_style=gnu-makefile
 short_desc="Utility for wildcard renaming, copying, etc"
```
Comment (maxice8) [inline srcpkgs/mmv/template]: GPL-2.0-only

File: srcpkgs/most/template [hunk 13]
```diff
@@ -1,16 +1,16 @@
...
-revision=6
+revision=7
 build_style=gnu-configure
 configure_args="--with-slang=${XBPS_CROSS_BASE}/usr"
 hostmakedepends="slang-devel"
 makedepends="slang-devel"
-short_desc="Paging program that displays, one windowful at a time, the contents of a file"
+short_desc="Paging program that displays, one windowful at a time, file contents"
 maintainer="Jens E. Becker <v2px@v2px.de>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/most/template]: GPL-2.0-or-later

File: srcpkgs/mozplugger/template [hunk 14]
```diff
@@ -1,13 +1,13 @@
...
+revision=4
 build_style=gnu-configure
+conf_files="/etc/mozpluggerrc"
 makedepends="libX11-devel"
 depends="m4"
-conf_files="/etc/mozpluggerrc"
 short_desc="Firefox plugin to embed arbitrary applications"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/mozplugger/template]: GPL-2.0-or-later

File: srcpkgs/perl-IPC-Run3/template [hunk 15]
```diff
@@ -1,15 +1,15 @@
...
+revision=2
+noarch=yes
 wrksrc="${pkgname#*-}-${version}"
 build_style=perl-module
 hostmakedepends="perl"
 depends="perl"
 short_desc='IPC::Run3 -  run a subprocess and redirect stdin, stdout, and/or stderr'
 maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
-homepage="https://metacpan.org/release/IPC-Run3"
 license="Artistic, BSD, GPL-1, GPL-2, GPL-3"
```
Comment (maxice8) [inline srcpkgs/perl-IPC-Run3/template]: Aristic-1.0-Perl, GPL-1.0-or-later

File: srcpkgs/polkit-qt5/template [hunk 16]
```diff
@@ -1,16 +1,16 @@
...
-revision=2
+revision=3
+wrksrc="polkit-qt-1-${version}"
 build_style=cmake
 hostmakedepends="pkg-config automoc4 git"
 makedepends="polkit-devel qt5-devel"
 short_desc="Qt-style PolicyKit API (Qt5)"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="https://projects.kde.org/projects/kdesupport/polkit-qt-1"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/polkit-qt5/template]: LGPL-2.1-or-later

File: srcpkgs/quimup/template [hunk 17]
```diff
@@ -1,20 +1,25 @@
...
+configure_args="INCLUDEPATH+='${XBPS_CROSS_BASE}/usr/include/qt5'
+ INCLUDEPATH+='${XBPS_CROSS_BASE}/usr/include/qt5/QtCore'
+ INCLUDEPATH+='${XBPS_CROSS_BASE}/usr/include/qt5/QtGui'
+ INCLUDEPATH+='${XBPS_CROSS_BASE}/usr/include/qt5/QtNetwork'
+ INCLUDEPATH+='${XBPS_CROSS_BASE}/usr/include/qt5/QtWidgets'"
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
 makedepends="libmpdclient-devel qt5-devel taglib-devel"
 short_desc="Client for MPD written in c++ and QT5"
 maintainer="beefcurtains <beefcurtains@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/quimup/template]: GPL-2.0-or-later

File: srcpkgs/schedtool/template [hunk 18]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=schedtool
 version=1.3.0
-revision=2
+revision=3
 build_style=gnu-makefile
 make_install_args="DESTPREFIX=/usr"
 short_desc="Query and set CPU scheduling parameters"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/schedtool/template]: GPL-2.0-only

File: srcpkgs/sopwith/template [hunk 19]
```diff
@@ -1,15 +1,15 @@
...
 version=1.8.4
-revision=1
+revision=2
 build_style=gnu-configure
+hostmakedepends="automake SDL-devel"
 makedepends="SDL-devel"
-hostmakedepends="automake ${makedepends}"
 short_desc="Sidescrolling shoot 'em up game"
 maintainer="beefcurtains <beefcurtains@users.noreply.github.com>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/sopwith/template]: GPL-2.0-or-later

File: srcpkgs/sysfsutils/template [hunk 20]
```diff
@@ -1,13 +1,13 @@
...
 version=2.1.0
-revision=6
+revision=7
 build_style=gnu-configure
 short_desc="System Utilities Based on Sysfs"
-homepage="http://linux-diag.sourceforge.net/Sysfsutils.html"
-license="GPL-2, LGPL-2.1"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-distfiles="${SOURCEFORGE_SITE}/linux-diag/$pkgname-$version.tar.gz"
+license="GPL-2, LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/sysfsutils/template]: GPL-2.0-only, LGPL-2.1-or-later

File: srcpkgs/wmctrl/template [hunk 21]
```diff
@@ -1,13 +1,13 @@
...
+revision=5
 build_style=gnu-configure
-short_desc="CLI tool to interact with EWMH compliant WMs"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
-license="GPL-2"
 hostmakedepends="pkg-config"
 makedepends="libXmu-devel glib-devel"
+short_desc="CLI tool to interact with EWMH compliant WMs"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/wmctrl/template]: GPL-2.0-or-later

File: srcpkgs/xdaliclock/template [hunk 22]
```diff
@@ -1,15 +1,15 @@
...
 version=2.43
-revision=2
+revision=3
 build_wrksrc=X11
 build_style=gnu-configure
 makedepends="libXext-devel libXt-devel"
 short_desc="Melting digital clock"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="MIT/X11"
```
Comment (maxice8) [inline srcpkgs/xdaliclock/template]: X11

File: srcpkgs/xbindkeys/template [hunk 23]
```diff
@@ -1,13 +1,14 @@
...
 version=1.8.6
-revision=2
+revision=3
 build_style=gnu-configure
 makedepends="pkg-config libX11-devel guile-devel guile gc-devel"
 depends="guile"
 short_desc="Launch shell commands with your keyboard or your mouse under X"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xbindkeys/template]: GPL-2.0-or-later

File: srcpkgs/xcb-util-image/template [hunk 24]
```diff
@@ -1,16 +1,16 @@
...
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config"
 makedepends="xcb-util-devel"
 short_desc="Utility libraries for XC Binding - Port of Xlib's XImage and XShmImage functions"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://xcb.freedesktop.org"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xcb-util-image/template]: X11

File: srcpkgs/xcb-util-keysyms/template [hunk 25]
```diff
@@ -1,16 +1,16 @@
...
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config"
 makedepends="libxcb-devel xcb-util-devel"
 short_desc="Utility libraries for XC Binding - Standard X key constants and conversion to/from keycodes"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://xcb.freedesktop.org"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xcb-util-keysyms/template]: X11

File: srcpkgs/xcb-util-renderutil/template [hunk 26]
```diff
@@ -1,16 +1,16 @@
...
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config"
 makedepends="libxcb-devel xcb-util-devel"
 short_desc="Utility libraries for XC Binding - Convenience functions for the Render extension"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://xcb.freedesktop.org"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xcb-util-renderutil/template]: X11

File: srcpkgs/xoris/template [hunk 27]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=xoris
 version=0.1e
-revision=2
+revision=3
 makedepends="libX11-devel"
 depends="rgb"
 short_desc="Grabs color from the screen and dumps it to stdout"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="MIT/X11"
```
Comment (maxice8) [inline srcpkgs/xoris/template]: X11

File: srcpkgs/xrootconsole/template [hunk 28]
```diff
@@ -1,13 +1,13 @@
...
-revision=2
+revision=3
 create_wrksrc=yes
 build_wrksrc="${pkgname}-${version%r*}"
 build_style=gnu-makefile
 makedepends="libX11-devel"
 short_desc="Fancy X console display"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xrootconsole/template]: GPL-2.0-or-later

File: srcpkgs/xteddy/template [hunk 29]
```diff
@@ -1,12 +1,12 @@
...
 version=2.2
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="imlib2-devel libXext-devel"
 short_desc="Your virtual comfort when things get rough"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xteddy/template]: GPL-2.0-or-later

File: srcpkgs/yasm/template [hunk 30]
```diff
@@ -1,13 +1,13 @@
...
 pkgname=yasm
 version=1.3.0
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="xmlto python"
 short_desc="Complete rewrite of the NASM assembler"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.tortall.net/projects/yasm/"
 license="BSD, Artistic, GPL-2, LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/yasm/template]: BSD-3-Clause, Artistic-1.0, GPL-2.0-or-later, LGPL-2.1-or-later

File: srcpkgs/xdaliclock/template (hunk 22 above)
Comment (Chocimier) [reply srcpkgs/xdaliclock/template]: It is not X11 too, but it is installed anyway.

**Comment by Chocimier** [comment]
Changed.

I forgot to pushed patches previously.

**Pull request closed without merging**